cache.js

```diff
@@ -254,8 +258,11 @@ function createCache(cache, message)
 //On cache creation request, find the data, record which models to watch, and send
 function primeCache(cache, app, message)
 {
-    const models = message.data.models;
-    const responseSubName = message.data.responseSubName;
+    message.ack()
+
+    const data = JSON.parse(message.data.toString());
```

Comment:
any reason we need to clone this? why not pass it plain like before?

Reply:
pubsub messages now must all be Buffers, so you have to toString() and JSON.parse to get the object/array data

Reply:
ohh I see, aight